src/index.html

```diff
@@ -10,8 +10,198 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>
+          How old are you?
+          <input 
+            type="number" 
+            name="age" 
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <br>
+        <label>
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+          >
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/230861420-2e47d0c9-08a7-432c-88d9-c64d45751868.png)

```suggestion
          <input type="date" name="date">
```